matching/LICENSE

```diff
@@ -0,0 +1,30 @@
+Copyright Author name here (c) 2018
```

Comment:
and this one.

Reply:
okay, but in that case they should be updated with actual content.

Reply:
I added our three names here, since we've touched the code in some manner or another (Radu first, of course). We'll just need to remember to update names if anyone else works on it.